app/uk/gov/hmrc/traderservices/views/DeclarationDetailsEntryView.scala.html

```diff
@@ -14,24 +14,24 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.traderservices.models.ConsignmentDetails
+@import uk.gov.hmrc.traderservices.models.DeclarationDetails
 
 @this(
     govukLayout: uk.gov.hmrc.traderservices.views.html.templates.GovukLayoutWrapper,
     forms: uk.gov.hmrc.traderservices.views.components.forms,
     html: uk.gov.hmrc.traderservices.views.components.html
 )
 
-@(form: Form[ConsignmentDetails], postAction: Call)(implicit request: Request[_], messages: Messages)
+@(form: Form[DeclarationDetails], postAction: Call)(implicit request: Request[_], messages: Messages)
```

Comment:
This view is not displaying correctly. 
<img width="439" alt="image" src="https://user-images.githubusercontent.com/55190161/94252936-842e1080-ff1c-11ea-9eff-6ff3118256fb.png">